data/src/main/scala/ch.epfl.scala.index.data/github/GithubDownload.scala

```diff
@@ -21,12 +21,17 @@ import play.api.libs.ws.ahc.AhcWSClient
 import play.api.libs.json._
 
 import scala.util._
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
 import java.nio.charset.StandardCharsets
 import java.nio.file.{Files, Path}
 
 import com.typesafe.config.ConfigFactory
 import org.slf4j.LoggerFactory
 
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
you should no import the global execution context.
import system.dispatcher at line 38.

more info here: http://doc.akka.io/docs/akka/current/scala/dispatchers.html